meta/boardreports/2023-02.md

```diff
@@ -0,0 +1,77 @@
+# InnerSource Patterns WG - Report for Board Meeting 2023-02
+
+## Meta
+
+* Reporting Period: November 1st, 2022 - January 31st, 2022
+* [merged PRs](https://github.com/InnerSourceCommons/InnerSourcePatterns/pulls?q=is%3Apr+closed%3A2022-11-01..2023-01-31+is%3Amerged)
+* [opened issues](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues?q=is%3Aissue+created%3A2022-11-01..2023-01-31+is%3Aopen)
+
+## Engagement
+
+The [patterns book] is the way InnerSource practices are captured and shared. Recent web analytics:
+
+* stable traffic on the patterns book (tracking_id: `G-QL1S8MW5D9`)
+  * 12,143 page views total (previous report was 12,627)
+  * ~135 per day
+* Most popular patterns:
+  * InnerSource Portal
+  * Core Team
+  * 30 Day Warranty
+  * Maturity Model
+  * Common Requirements
+* traffic for translations:
+    * Japanese - No significant traffic yet.
+    * Chinese - No stats yet.
+
+## Changes
+
+Changes are contributed via the [InnerSourcePatterns] repository:
+
+* [Chinese book](https://patterns.innersourcecommons.org/v/zh/) release in 2023/01
+* Issues with getting Japanese content indexed by Google and other search engines has been fixed (by gitbook)
+* New **Structured** patterns:
+    * [Extensions to Manage Contributions at Scale](https://patterns.innersourcecommons.org/p/extensions-for-sustainable-growth) - great work @Ssukriti
+* New **Initial** patterns:
+    * [Incubator Pipeline](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/incubator-pipeline.md) - thanks @robtuley for moving this long-standing PR along and getting the content into our repo for greater visibility of the ISC community
+    * [InnerSource Customer Interview Questions](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/innersource-customer-interview-questions.md) - contributed by @kschueths as part of the last Summit. Thank you!
+* Adding Known Instances to various patterns e.g. WellSky, Dazn, and Mercado Libre. Special thanks to @privero-meli from our community in Brazil for adding Mercado Libre.
+* We are trying out GitHub Releases ([example](https://github.com/InnerSourceCommons/InnerSourcePatterns/releases/tag/v1.4)) to highlight all contributors to our book. We hope that they will appreciate that acknowledgement, and continue their contributions.
+* Various minor updates to existing patterns.
+
```

Comment:
I don't know if this is the right place to write this, but I'll do it for the record.
```suggestion
* Recorded 20 instructional YouTube videos on InnerSource Patterns in Japanese. All will be rolled out in Q1 and Q2 of CY2023. It is hoped that the InnerSource Patterns page and the Japanese page will benefit from improved SEO.

```

Reply:
This is great!
Which link can I use to point the "20 instructional YouTube videos on InnerSource Patterns in Japanese"?

Reply:
I think I found the playlist, will add it to the report.
https://www.youtube.com/watch?v=rbfXGedu81g&list=PLzsDnO3v-utv349AtbxElYAqrXN6XZVqR

However note that YouTube shows me "12 unavailable videos are hidden". 
Probably those 12 of the 20 videos are not made public yet?

Reply:
My bad, I did not read your statement correctly
> All will be rolled out in Q1 and Q2 of CY2023

Still odd that YouTube tells me that there are more videos in the playlist than what I can actually watch :)